geometry_msgs/PointStamped gas_sensor_position
float64 rate
---
bool success